mongodb/driver/query/construct.xml

```diff
@@ -42,7 +42,7 @@
          <row>
           <entry>limit</entry>
           <entry><type>integer</type></entry>
-          <entry>Client side maximum number of documents to return</entry>
+          <entry>Maximum number of documents to return to the client</entry>
```

Comment:
Thats an implementation detail that may be skewed next week, next month, or 6months from now.
Besides, the cursor is still fine, alive and kicking, on the server after having sent back these amount of documents.
We need to manually kill it. It doesn't automatically selfdestruct.

I read your change as there only will be X entries in the cursor created by the query, and the server will not and cannot send more results. That isn't true. That works with negative limits though...... but thats different, it will only be one batch (and if it doesn't fit in the batch, you won't see the rest of the results)